vexim/siteaddsubmit.php

```diff
@@ -138,26 +148,10 @@
                 "&type={$_POST['type']}");
         die;
       }
-    } else {
-      header ("Location: site.php?failaddeddomerr={$_POST['domain']}");
-      die;
-    }
-  } else if ($_POST['type'] == "alias") {
-    $query = "INSERT INTO domainalias (domain_id, alias)
-              SELECT domains.domain_id, :alias FROM domains WHERE domains.domain_id=:domain_id";
-    $sth = $dbh->prepare($query);
-    $sth->execute(array(':domain_id'=>$_POST['aliasdest'], ':alias'=>$_POST['domain']));
-    if ($sth->rowCount()!==1) {
-      header ("Location: site.php?failaddeddomerr={$_POST['domain']}");
-      die;
-    } else {
-      header ("Location: site.php?added={$_POST['domain']}" .
-              "&amp;type={$_POST['type']}");
-      die;
-    }
-  } else {
-    header ("Location: site.php?failaddedpassmismatch={$_POST['domain']}");
+    } 
   }
+  header ("Location: site.php?failaddeddomerr={$_POST['domain']}");
+  die;
```

Comment:
I don't think we can ever reach these two lines. And having `die()` as the last PHP call in a file which is not `include()`d is also rather pointless.